.gitignore

```diff
@@ -6,4 +6,4 @@ coverage.lcov
 package-lock.json
 yarn.lock
 es
-lib
\ No newline at end of file
+lib/*
```

Comment:
Good catch, please make them:
```
/es
/lib
```
To be explicit.

Reply:
done